Fixes Yeosa eyes not glowing, fixes Unathi having appendixes, fixes Unathi eyes being slightly lighter but only from one side

tiramisuapimancer opened the pull request:
This was originally supposed to be a PR about making Yeosa eyes glow like they were intended to in the PR that added them, but along the way I found two other minor issues with Unathi that I decided to fix while I was there (because I have no idea what I'm doing, and I was changing these things to see if they were the reason the eyes weren't working). Didn't see anything in the contribution guidelines against wildly unatomized PRs, so here we are.

![image](https://user-images.githubusercontent.com/32314478/130359741-897178e4-1814-4f56-a650-10d619546956.png)

Restructures Unathi organ list/override so that they don't have appendixes, and so that Yeosa don't have both a regular head and a Yeosa head 

Unathi eyes were a slightly different shade in a random side state so while I was there I fixed that

:cl: 
bugfix: unathi do not have appendixes as they were never intended to have appendixes
bugfix: yeosa'unathi eyes glow like they were intended to
/:cl:

tiramisuapimancer committed:
gatorade 500u

tiramisuapimancer commented:
Closes #31033

SierraKomodo reviewed (dismissed):
According to icondiff the .dmi file change also removed the state name from the sprite.

tiramisuapimancer commented:
> 
> 
> According to icondiff the .dmi file change also removed the state name from the sprite.

That's actually the only way I could see to make the eye-glowing work. 
The way glowing eyes code works right now is that any icons with state names aren't considered for being on the unaffected-by-light plane. This made me nearly cry when I figured it out, and I hate it. Also, I don't know how to recode the glowing eyes bit in the head organ .dm to make this not the case. I'm so sorry.
(Everything I've seen so far suggests that leaving the eyes state-name-less leaves them working and doesn't actually break anything, because that's how adherent and human eyes are, and them being that way is what lead to me figuring this out- I made Unathi have human eyes and Yeosa eyes properly glowed, so I checked Adherent's eyes .dmi and saw they had no state name, removed Unathi eyes state name, everything worked fine.)

Spookerton reviewed (approved):
i assume it functions
